shared/src/bundles/users/types/user-sign-in-response-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type UserSignUpResponseDto } from './user-sign-up-response-dto.type.js';
+
+type UserSignInResponseDto = UserSignUpResponseDto;
```

Comment:
In Figma design SignUp form also contains a name field. So, this way you have to add name filed to the `UserSignUpResponseDto` and then use it something like:
`type UserSignInResponseDto = Omit<UserSignUpResponseDto, 'name'>`.

Reply:
The signUp method was already created before me, I made small changes to it (creating a token and minor validation). I'm not sure if this applies to my assignment. But if it is then I shouldn't write like this `type UserSignInResponseDto = UserSignUpResponseDto;` because in a signIn we shouldn't return fullName

Reply:
You are right, it's not your task, but since you've started implementing it I've commented on it. But if I create a separate branch from your branch to finish the implementation of sign-up functionality I think we should discuss this to avoid further misunderstandings. Or you can just review my PR once I create it